src/lib/expansion/expansion.spec.ts

```diff
@@ -0,0 +1,69 @@
+import {async, TestBed} from '@angular/core/testing';
+import {Component} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {MdExpansionModule} from './index';
+
+
+describe('MdExpansionPanel', () => {
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [MdExpansionModule],
+      declarations: [
+        PanelWithContent
+      ],
+    });
+    TestBed.compileComponents();
+  }));
+
+  it('should apply expanded classes as appropriate', () => {
```

Comment:
I would just make this `should expand and collapse the panel` with tests that check the css class and the absence/presence of the content.